GoogleChromeLabs/wp-sitemaps
# Try removing stylesheets completely

### Description

Completely removes the default sitemap xml stylesheet but keeps the filters to allow adding one.

### Type of change
Please select the relevant options:


### Steps to test

1. Open sitemap / sitemap index
1. Verify that no stylesheet is being added

### Acceptance criteria
